lib/builder.js

```diff
@@ -141,6 +155,14 @@ function addExtraOutputs(output, tree, opts) {
 Builder.prototype.buildTree = function(tree, outFile, opts) {
   var loader = this.loader;
   var self = this;
+
+  // allow passing opts
+  // as second argument.
+  if(isPlainObject(outFile)){
```

Comment:
I don't believe that would be correct - when callingl `new Builder(tree, null, {"real": "options})` that would take `opts` as `{}`.